sandbox/src/pages/GridPage.tsx

```diff
@@ -0,0 +1,251 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: Apache-2.0 */
+
+import { Grid } from "core-ui/core/Grid";
+import { CodeDrawer } from "../components/CodeDrawer";
+import { Separator } from "core-ui/core/Separator";
+
+export const GridPage = () => {
+  const separatorStyle = {
+    border: "0.1rem dashed #000",
```

Comment:
Just a nit so we can see these beautiful grid segments on dark mode.
```suggestion
    border: "0.1rem dashed var(--border-secondary-color)",
```